lib/Connector/ConnectorInterface.php

```diff
@@ -52,4 +52,5 @@ public function isProviderSetting($setting): bool;
     public function getSettingsFormTwig(): string;
     public function getSettingsFormJavaScript(): string;
     public function processSettingsForm(SanitizerInterface $params, array $settings): array;
+    public function getAvailablePartners(bool $isThrowError = false, ?string $withApiKey = null);
```

Comment:
This method is specific to the SSP connector and wouldn't be applicable to all connectors, so it isn't appropriate to add it to the interface.

However, we can use the proxy route to add a method specifically to our SSP connector. Have a look at the `connector.edit.form.proxy` route in routes-web.